src/UI/Radio.elm

```diff
@@ -4,6 +4,7 @@ module UI.Radio exposing
     , withButtons, withSelected
     , RadioWidth, withWidth, widthFull, widthRelative
     , renderElement
+    , horizontalDisposition, withDisposition
```

Comment:
I would add a `verticalDisposition` as well, as IMO all the "options" should be available even the default one.